scapy/route.py

```diff
@@ -21,8 +22,11 @@
 
 class Route:
     def __init__(self):
-        self.resync()
-        self.cache = {}
+        if isinstance(conf.route, list):
+            self.routes = conf.route
+        else:
+            self.resync()
+        self.invalidate_cache()
```

Comment:
I'm not very happy with that part. Is it really a good way of storing a temp cache ?

Reply:
And what's the role of this change in the context of whole MR? Especially that this code is shared (not windows specific)...

Reply:
The role of this change is to use routes that have been loaded in arch/\_\_init\_\_.py (where it was using POWERSHELL_MANAGER). If we move to atexit, this will be obsolete.

Reply:
Actually atexit is so hard to setup on IPython :/